index.html

```diff
@@ -30,7 +32,25 @@
         <div class="desktop-menu">Contact</div>
         <div><img src="assets/images/post.png" alt="Mail Box"></div>
       </div>
-
+      <div class="popup" id="popup-1" >
+        <div class="close-button-container">
+          <button type="button"  class="close-button" onclick="togglePopup()"> &times;</button>
+        </div>
+       
+         
+          <div class="mobile-menu" onclick="togglePopup()" > <a href="#middlesec">Portfolio</a> </div>
+          <hr>
+          <div class="mobile-menu" onclick="togglePopup()"> <a href="#aboutme">About</a>  </div>
+          <hr>
+          <div class="mobile-menu" onclick="togglePopup()"> <a href="#Contact-me">Contact</a> </div>
+          <hr>
+          <div class="drop-bottom">
+            <hr>
+          </div>
+       
+       
+        
+ </div>
```

Comment:
- [ ] Kindly make sure that the modal is taking the entirety of the screen's width and height.
- [ ] Please make sure that the user can't scroll to the page's content while the modal is open (See GIF below). (You can use `position: fixed`)

![chrome-capture](https://user-images.githubusercontent.com/62290545/162825636-92ccc0c5-7861-4ac8-bcee-fb4de6ebce41.gif)